cluster/addons/cluster-monitoring/influxdb/heapster-controller.yaml

```diff
@@ -84,6 +84,8 @@ spec:
             - --container=heapster
             - --poll-period=300000
             - --estimator=exponential
+          securityContext:
+            runAsNonRoot: false
```

Comment:
this is weird... is this component expecting to run as root? if so, setting `runAsUser: 0` seems way more straightforward.